### DISCUSSION TITLE
The interface for getting order details did not return the address. Please check whether it is normal. The address information exists in the Etsy backend.

### DISCUSSION THREAD

**Discussion by HG-93** [etsy/open-api#1313] [Q&A]
The interface for getting order details did not return the address. Please check whether it is normal. The address information exists in the Etsy backend.

url：[https://openapi.etsy.com/v3/application/shops/54915925/receipts/3457319585](https://openapi.etsy.com/v3/application/shops/26575409/receipts)

response：{"receipt_id":3457319585,"receipt_type":0,"seller_user_id":983843601,"seller_email":"chichishopfour@gmail.com","buyer_user_id":977417473,"buyer_email":"stephaniebai@icloud.com","name":"Stephanie Mahoney","first_line":null,"second_line":null,"city":null,"state":null,"zip":null,"status":"Paid","formatted_address":null,"country_iso":null,"payment_method":"cc","payment_email":null,"message_from_payment":null,"message_from_seller":null,"message_from_buyer":"","is_shipped":false,"is_paid":true,"create_timestamp":1729778085,"created_timestamp":1729778085,"update_timestamp":1729778093,"updated_timestamp":1729778093,"is_gift":false,"gift_message":"","gift_sender":"","grandtotal":{"amount":2747,"divisor":100,"currency_code":"USD"},"subtotal":{"amount":2040,"divisor":100,"currency_code":"USD"},"total_price":{"amount":5099,"divisor":100,"currency_code":"USD"},"total_shipping_cost":{"amount":579,"divisor":100,"currency_code":"USD"},"total_tax_cost":{"amount":128,"divisor":100,"currency_code":"USD"},"total_vat_cost":{"amount":0,"divisor":100,"currency_code":"USD"},"discount_amt":{"amount":3059,"divisor":100,"currency_code":"USD"},"gift_wrap_price":{"amount":0,"divisor":100,"currency_code":"USD"},"shipments":[],"transactions":[{"transaction_id":4304751919,"title":"5&quot; Live Succulent Plants, Cotyledon Orbiculata Variegated, Easy Care Plants","description":"Plant Size:  5 inches\n\nYou&#39;ll getting beautiful rare succulent plants.\n\nThey will ship bare root without pot and soil.\n\nI will package with extreme care and lots of love.\n\nCaring For succulent:\n\n1.Every 10 to 15 days during the first month or two. Once it is rooted properly, water it 2 to 3 times in the summer. Avoid overwatering.\n\n2.They undergo aestivation when temperatures reach above 35\u00b0C, their leaves will curl to prevent excessive water loss and sunburn, giving them a rose-like shape.Give minimal water when they are dormant during summer.\n\n3.The active growing season of succulent occurs between 18-24\u00b0C. After dormancy, the leaves will open up, drench the soil thoroughly once it is dry.\n\n4.Succulent plants need well-drained soil including gravelly or sandy. Avoid heavy clay as it will destroy the succulent.\n\n5.Do NOT water directly onto the greenovia as the water will be trapped in between the leaves, causing rot.\n\n\nKind tips:\n\n1.The plant you receive may not be the same color or shape as the picture. The color of the plants will fade during transit which is normal. But they will recover to their beauty in a sunny, ventilatedenvironment and through good care.\n\n2.In order to ensure the survival rate of succulents, we always dry them for delivery, so it is normal to receive wilted and become slightly smaller sizes. Please plant in wet soil, it will recover in a week or two.\n\n3.Some succulent leaves are relatively fragile, and it is normal for the leaves to fall off. Put the leaves flat on the surface of the soil. When the temperature and humidity are suitable, the leaves will germinate. When the cuttings grow up, they will become a new succulent again.","seller_user_id":983843601,"buyer_user_id":977417473,"create_timestamp":1729778085,"created_timestamp":1729778085,"paid_timestamp":1729778091,"shipped_timestamp":null,"quantity":1,"listing_image_id":6361567464,"receipt_id":3457319585,"is_digital":false,"file_data":"","listing_id":1797968908,"sku":"NY098","product_id":22984962128,"transaction_type":"listing","price":{"amount":5099,"divisor":100,"currency_code":"USD"},"shipping_cost":{"amount":0,"divisor":100,"currency_code":"USD"},"variations":[],"product_data":[],"shipping_profile_id":245427832126,"min_processing_days":5,"max_processing_days":7,"shipping_method":null,"shipping_upgrade":null,"expected_ship_date":1730437140,"buyer_coupon":30.589999999999999857891452847979962825775146484375,"shop_coupon":30.589999999999999857891452847979962825775146484375}],"refunds":[]}

X-Request-ID : EuOO12FbofOqrsaXM2FZnZ6Wiyc3
![image](https://github.com/user-attachments/assets/0482c421-8c04-4db0-b868-60bf81d6d32d)

**Comment by dobbamca** [comment]
https://github.com/etsy/open-api/releases

**Reply by Clarry100** [reply]
I hope this change does not apply here in Australia. 
We are a store located in Australia and ONLY selling in the Australian market. 
I am currently engaged in trying to get this exact data from etsy via the same API endpoint, rather than have to manually click buttons/links on the main site for the same info.
The release notes say USA, but unsure.

**Reply by HG-93** [reply]
That is to say, if you need to obtain address information, you need to be an Etsy preferred partner or belong to an approved category, right? Then why can some orders in the same store obtain addresses while others cannot?

**Reply by Norithiel** [reply]
I found this thread once my Make scenario, which was syncing Etsy orders to my Airtable base, stopped working. I also receive "nulls" for all the address-related fields. Also Make support specified that my scenario is hosted in the EU1 server, and my store is also non US-based.

**Comment by dpl-subash** [comment]
It says existings sellers will not effect the changes but we are not able to get formatted address from the API. 
Also how can we apply for Etsy preffered partner or approved category for our application?
@nateswart

**Comment by laloutre87** [comment]
as if removing the email  wasn't bad enough, now we can't access the address  all together ?  Right on  christmas shipping time.  Who make  this kind of decisions...

**Comment by qq2161811** [comment]
I also encountered the same problem. These basic and important information cannot be obtained. What is the use of these APIs? They are not friendly to developers at all. If you are afraid of information leakage, you can do permission approval and warnings. I think this is okay, instead of not being able to obtain information directly, which makes many tasks impossible to complete.
